includes/backends.inc

```diff
@@ -31,8 +31,8 @@ EOQ;
 
     // Sort the objects into their proper order.
     $sort = function($a, $b) {
```

Comment:
function `intval` does not return empty values (only 0 on error), so why check for empty values?
Also, why check for equality (`if ($a === $b) { return 0; }`) when `return $a - $b` accomplishes the same.
Question: is it possible that `$a['seq']['value']` does not exist? If so, shouldn't it be checked to silence warnings?